### PR TITLE
ENHANCEMENT Enable UploadField.js within non-react sections

### PR DESCRIPTION
**Pull request by tractorcow**
Required for https://github.com/silverstripe/silverstripe-asset-admin/pull/331

### CODE REVIEW

File: admin/client/src/boot/index.js [hunk 1]
```diff
@@ -68,6 +68,10 @@ function appBoot() {
   // Bootstrap routing
   const routes = new BootRoutes(store);
   routes.start(window.location.pathname);
+
+  // @TODO - Remove once we remove entwine
+  // Enable top-level css selectors for react-dependant entwine sections
+  window.jQuery('body').addClass('react-boot');
```
Comment (dhensby) [inline admin/client/src/boot/index.js]: please can we use a `.js-` prefix on this?

http://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/#javascript-namespaces-js-

Comment (flamerohr) [inline admin/client/src/boot/index.js]: I don't think it's necessary for this instance, seems to act more like a modernizr for react booting.

Although a check for `window.jQuery` existing would be nice.

**Comment by tractorcow** [comment]
Feedback implemented

**Comment by dhensby** [comment]
NPM failure pre-existing.

**Pull request merged**